roles/common/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: restart sensu-client
+  service:
+    name: sensu-client
+    enabled: yes
```

Comment:
Let's keep it more explicit here. I suggest we skip ``enabled: yes`` and just use ``state: restarted``.